chart-app/src/components/ChartApp.vue

```diff
@@ -83,22 +94,13 @@ a {
   margin: 40px 0 0;
 }
 
```

Comment:
```suggestion
.hidden {
    display: none;
}
```
Here I'm creating a custom class to hide things, instead of using the css `hidden` attribute - this is quite common in Vue, and I'm sure we do it a few times in simoc-web. The benefit is that, with classes, you can add conditional classes in the template - like I did above.